fieldagent_access_token.txt.example

```diff
@@ -0,0 +1 @@
+<Your FieldAgent access token goes here>
```

Comment:
Can you an example token here?

Reply:
How's this? https://github.com/SenteraLLC/file-upload-examples/pull/2/commits/2efaddf02fe97a919828a444772e7861ba91f626